test/EventDriven.EventBus.Abstractions.Tests/EventBusTests.cs

```diff
@@ -9,11 +9,13 @@ namespace EventDriven.EventBus.Abstractions.Tests
     public class EventBusTests
     {
         [Theory]
-        [InlineData(TopicType.Implicit, null)]
-        [InlineData(TopicType.Implicit, "v1")]
-        [InlineData(TopicType.Explicit, null)]
-        [InlineData(TopicType.Explicit, "v1")]
-        public async Task EventBus_Should_Invoke_Event_Handlers(TopicType topicType, string prefix)
+        [InlineData(TopicType.Implicit, null, null)]
+        [InlineData(TopicType.Implicit, "v1", null)]
+        [InlineData(TopicType.Explicit, null, null)]
+        [InlineData(TopicType.Explicit, "v1", null)]
+        [InlineData(TopicType.Explicit, null, "suffix")]
+        [InlineData(TopicType.Explicit, "v1", "suffix")]
```

Comment:
Copy lines 16 and 17 to the `EventBus_Should_Remove_Event_Handlers` test on line 52.